nanoFramework.Logging.Syslog/Facility.cs

```diff
@@ -0,0 +1,42 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace nanoFramework.Logging.Syslog
+{
+	/// <summary>
+	/// Available facility types (RFC 3164).
+	/// </summary>
+	public enum Facility
```

Comment:
Wouldn't it be beneficial to have Intellisense comments here?
Or maybe just on the one(s) that this library supports?

On a second thought it could be even better to have here only the supported ones...

Comments?

Reply:
`Facility` is an arbitrary value. Any of them can be used and is normally defined in the call to `SysLoggerFactory` constructor. For `Severity` effectively the mapping from `LogLevel` to Severity limits it to some of them. But You can always use `SysLogger.Client.SendMessage` methods if you want to use the severities that don't directly map to a `LogLevel` value.